int/api/html/front/event-manager.js

```diff
@@ -11,7 +11,7 @@ class EventManager {
             })
             .catch((e) => {
                 // TODO Implement retry mechanism here
-                console.error(e);
+                console.error(e.response.data);
```

Comment:
And what happens if the error is not in a server response?
You will get an exception 'e.response' is not defined...
You can find in the code example [here](https://axios-http.com/docs/handling_errors) different keys to get according to the different scenarios.
I don't know if it's exhaustive though...